README.md

```diff
@@ -1 +1,24 @@
-# blog-app rails
\ No newline at end of file
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
```

Comment:
- [ ] **Required:** Please update your README using the Microverse given template [here](https://github.com/microverseinc/readme-template).